ports/libevent/portfile.cmake

```
@@ -38,20 +42,27 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-if (VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/libevent)
-else ()
-    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake TARGET_PATH share)
-endif()
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake TARGET_PATH share)
 
 file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/libevent/)
 file(RENAME ${CURRENT_PACKAGES_DIR}/bin/event_rpcgen.py ${CURRENT_PACKAGES_DIR}/tools/libevent/event_rpcgen.py)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share)
 if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
     file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
 endif()
 
+set(_target_suffix)
+if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
+    set(_target_suffix static)
+else()
+    set(_target_suffix shared)
+endif()
+file(READ ${CURRENT_PACKAGES_DIR}/share/libevent/LibeventTargets-${_target_suffix}.cmake _content)
+string(REPLACE "${CURRENT_PACKAGES_DIR}" "${CURRENT_INSTALLED_DIR}" _content "${_content}")
+file(WRITE ${CURRENT_PACKAGES_DIR}/share/libevent/LibeventTargets-${_target_suffix}.cmake "${_content}")
```

```suggestion
vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/share/libevent/LibeventTargets-${_target_suffix}.cmake
 "${CURRENT_PACKAGES_DIR}" 
 "${CURRENT_INSTALLED_DIR}"
  )
```
Could you please use `vcpkg_replace_string()` to replace a string in a file?